media/css/firefox/new/batm-machine.scss

```diff
@@ -0,0 +1,535 @@
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+@import '../../pebbles/includes/lib';
+
+
+@font-face {
+    font-family: 'Droid Sans Mono';
+    font-weight: normal;
+    font-style: normal;
+    src: url('/media/fonts/DroidSansMono.woff2') format('woff2'),
+         url('/media/fonts/DroidSansMono.woff') format('woff');
+}
+
+@keyframes fade-in {
+    0% {
+        opacity: 0;
+        transform: translateY(50px);
+    }
+    100% {
+        opacity: 1;
+        transform: translateY(0);
+    }
+}
+
+html,
+body {
+    background: #ec8c32;
+    background: linear-gradient(135deg, #ec8c32 0%,#e5472c 100%);
+    min-height: 100%;
```

Comment:
This isn't new to these variations, but I notice there's a 2px solid white top border on the pages left from the old tabzilla. Probably good to remove?